build/secondary/testing/gtest/BUILD.gn

```diff
@@ -18,7 +18,10 @@ config("gtest_config") {
   ]
 
   # Gtest headers need to be able to find themselves.
-  include_dirs = [ "include" ]
+  include_dirs = [
```

Comment:
in general i think i'd prefer to keep this aligned with upstream if it's functional